openacademy/views/openacademy_session_board_view.xml

```diff
@@ -0,0 +1,61 @@
+<odoo>
```

Comment:
I am installing your module using these changes, this view is giving me a traceback

![image](https://user-images.githubusercontent.com/11479473/171493115-0d3ae817-f59b-4515-be03-0bc5f60d6877.png)


Could you check it please?

Reply:
Maybe is my Odoo version, it's okay. I trust you 😄